src/classes/STG_SettingsManager_TEST.cls

```diff
@@ -1335,4 +1724,123 @@ private with sharing class STG_SettingsManager_TEST {
         Test.stopTest();
         system.assertEquals(true, panel.isRunningBatch);
     }
+
+    //Helper Methods
+    /**
+     * @description Builds a read only user with readonly profile
+     * @return User
+    */
+    private static User createReadOnlyUser() {
+        User usr = UTIL_UnitTestData_TEST.createUser(UTIL_UnitTestData_TEST.PROFILE_READONLY_USER);
+        return usr;
+    }
+
+    /**
+    * @description create a instance of opportunity naming settings
+    * @return opportunity naming settings
+    */
+    private static Opportunity_Naming_Settings__c setUpOppNamingSettings() {
+        String optionName = STG_PanelOppNaming_CTRL.AttributionOptions.Contact_Donations.name();
+        OPP_OpportunityNaming_TEST.setUpOppNamingSettings(new Map<String, Object>{ 'Attribution__c' => optionName });
+        return UTIL_ListCustomSettingsFacade.getMapOppNamingSettings().values()[0];
```

Comment:
Option: can make change to `OPP_OpportunityNaming_TEST.setUpOppNamingSettings` to return the settings instead of getting querying and getting the first one.